nailgun/entities.py

```diff
@@ -2267,9 +2268,11 @@ class ContentView(
 
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
+            'auto_publish': entity_fields.BooleanField(),
             'component': entity_fields.OneToManyField(ContentViewVersion),
             'composite': entity_fields.BooleanField(),
             'content_host_count': entity_fields.IntegerField(),
+            'content_view_components': entity_fields.OneToManyField(ContentViewComponent),
```

Comment:
one last nitpick from me (sorry for not noticing earlier): i'd prefer `content_view_component` since it's `OneToManyField` so it's gonna be turned into plural automatically. Just for sake of consistency.